src/DatePicker/test/ToolbarSpec.tsx

```diff
@@ -2,16 +2,27 @@ import React from 'react';
 import ReactTestUtils from 'react-dom/test-utils';
 import { render, screen } from '@testing-library/react';
 import sinon from 'sinon';
-import { getDOMNode } from '@test/testUtils';
+import { testStandardProps } from '@test/commonCases';
 
 import Toolbar from '../Toolbar';
 
 describe('DatePicker - Toolbar', () => {
+  const fontSize = '12px';
+  testStandardProps(
+    <Toolbar
+      calendarDate={new Date(2021, 11, 24)}
+      className="custom"
+      classPrefix="custom-prefix"
+      style={{ fontSize }}
+      locale={{}}
+    />
+  );
```

Comment:
Setting `fontSize` does not seem necessary here. It's included in `testStandardProps` cases.